config/local-certs/rootCA.csr.cnf

```diff
@@ -0,0 +1,12 @@
+[req]
+distinguished_name = req_distinguished_name
+prompt = no
+
+[ req_distinguished_name ]
+countryName = US
+stateOrProvinceName = District of Columbia
+localityName = Washington
+organizationName = login.gov development
+organizationalUnitName = identity-pki
+emailAddress = noreply@login.gov
+commonName = identity-pki Development Certificate
```

Comment:
This is the name that shows up in the Keychain Access app, I think it's easily identifiable for what it is? But it's long enough to get truncated, so if anybody has suggestions for what to call this thing, I'm all ears!

It's important to make it clear what it is because for the time being we still need to go in the Keychain Access UI, find this thing, and mark it as trusted (or delete it when recreating)